### ISSUE TITLE
Mark up mobile "Menu" button accessibly

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#844]
**Page 98 in DAC report**

**Issue from DAC report:**
The Menu is focusable, but not keyboard actionable. It has no role, and nothing to describe its expanded or collapsed status to screen reader users. It has been implemented as a <span> which has been assigned the role of ‘navigation’ region.

The search button is not focusable or keyboard actionable and has no role nor expanded or collapsed status. 

**Solution from DAC report:**
Buttons should be marked up as such to ensure that they have a properly associated role to expose their behaviour and purpose to assistive technology. The aria-expanded attribute should be associated with the button. The navigation role associated with the ‘menu’ should be associated with the navigational region it triggers.

**Example from DAC report:**

```
<ul class="top-level">
    <li id="menuToggle" aria-label="Menu toggle. Click to expand or collapse 
      the menu." aria-haspopup="true">
        <button data-target="menu" aria-expanded="true">Menu</button>
    </li>
    <li><button data-target="search" aria-expanded="false">Search</span> 
    </button>
</ul>
…
<div role="navigation" aria-label="Menu">
    <ul class="nav navbar-nav menu-target contrast-switcher" id="menu"
    style="display: block;">…/ul>
</div>
```
